lib/bugsnag.rb

```diff
@@ -17,6 +17,8 @@
 # as it doesn't auto-configure when loaded
 require "bugsnag/integrations/rack"
 
+require "bugsnag/utility/circular_buffer"
```

Comment:
In your design doc, I thought you were planning to directly use `Bugsnag::Utility::CircularBuffer` in bugsnag/configuration.rb. If that's the case, you should `require` it in that file instead. However, since you're not using the `CircularBuffer` class yet, I would just not require it until you reach the pull request where you use it.

Reply:
Agreed